node/chainSimulator/components/stateComponents.go

```diff
@@ -31,7 +31,7 @@ type stateComponentsHolder struct {
 }
 
 // CreateStateComponents will create the state components holder
-func CreateStateComponents(args ArgsStateComponents) (factory.StateComponentsHolder, error) {
+func CreateStateComponents(args ArgsStateComponents) (factory.StateComponentsHandler, error) {
```

Comment:
Please rename to `CreateStateComponents` and remove the inner closeHandler declaration & usage

Reply:
done